### PR TITLE
Update ACT setup guide

### PR DESCRIPTION
**Pull request by xpdota**
* Added instructions for using OP's encounter end settings
* Added note that the number displayed in the image for the "Get Plugins" window is not the current one

### CODE REVIEW

**Comment by xpdota** [comment]
https://github.com/xpdota/docs/commit/b9a9ce6689b426f2d18793d6b678e6af0ed5c134 😠

Github editor adds the EOL at EOF no matter what I do, will check out locally and fix it if you want me to.

File: setup/README.md
Comment (quisquous) [inline setup/README.md]: Can you check these images into the repo?

File: setup/README.md
Comment (quisquous) [inline setup/README.md]: Would it be worth recommending just unclicking this box entirely?

File: setup/README.md
Comment (quisquous) [inline setup/README.md]: Would you mind updating this screenshot to reflect the changed id?

**Review by quisquous — COMMENTED** [review]
Thanks for all the doc improvements!

File: setup/README.md
Comment (xpdota) [reply setup/README.md]: Done

File: setup/README.md
Comment (xpdota) [reply setup/README.md]: Hmm, I don't know. I think it's still worth having some kind of value there for fresh patches when OP's memory sigs might not be updated yet.

Comment (quisquous) [reply setup/README.md]: I sorta disagree.  Most of the time it'll break on wipes which come from ACT log lines, which is good enough for most use cases?  I've had it unchecked for a very long time with no issues.

Comment (quisquous) [reply setup/README.md]: Mostly I also mention this because this is also Aho's advice in #general pins: https://cdn.discordapp.com/attachments/551474815727304706/921609700842299413/Step1.PNG

So, I figure it makes sense for the setup guide to match the pins? Or we change both of them?

Comment (xpdota) [reply setup/README.md]: I guess I can just mention both ways of doing it

File: setup/README.md
Comment (xpdota) [reply setup/README.md]: Done

File: setup/README.md
Comment (quisquous) [reply setup/README.md]: You're saying that you disagree that the setup guide and the pins should have the same recommendation?

Comment (xpdota) [reply setup/README.md]: Well, I didn't write the pin, so I can't vouch for the information.

There are a few situations where you're still "in combat" but it shouldn't be considered the same pull, e.g. Praetorium when people tag mobs and then hit the teleporter (unsure if it happens with the remade version). The ACT timer is more of a failsafe for those situations. 

Also, more generally, I think it's fine for a pin to be the short version, and then the guide can elaborate on it further.

**Comment by Makar8000** [comment]
I know the OP maintainers have been busy lately, but when any of you get the chance could you take another look at this PR?

I noticed in the FFLogs help article they reference the old ACT Guide (the one last updated in 2020) and figured I would ask them to use this one instead. Before that, it would be nice to have this one updated.

File: setup/README.md
Comment (MyTechnoHunter) [inline setup/README.md]: it no longer has [FFXIV+others] in front of the plugin in the list

**Review by MyTechnoHunter — CHANGES_REQUESTED** [review]

File: setup/README.md
Comment (quisquous) [reply setup/README.md]: Belatedly, I don't think there's any vouching required here.  The pin says:

```
About : Handling ACT Encounters

Here's the recommendation : 
First, turn off that option in ACT Settings. 
>Then, you can either let OverlayPlugin handle it
>Or Triggernometry (Misc repo)
>Or you can make an in-game macro (but you'll have to press it everytime)
```
image
(plus some other images about how to have trigg or overplayplugin handle it)

Belatedly, I remember that 60 seconds is not enough for some fight cutscenes, which is why I also think "don't click the checkbox" is a better recommendation.  I also continue to think that the pin in the discord and this guide should have the same advice to be less confusing to users.

**Comment by quisquous** [comment]
Sorry I forgot about this review.  Please ping me if needed.

**Comment by xpdota** [comment]
Made lots of changes, please take a look.

File: setup/README.md
Comment (quisquous) [inline setup/README.md]: The text "set it to a relatively large value" plus a screenshot that suggests "6" is very misleading.  I continue to think it's better to give the same advice that's in discord pins for consistency.  There have been a lot of questions around this over time.

```suggestion
To do this, first find ACT's encounter split timeout (Options > Main Table/Encounters > General).
It is recommended to disable it entirely by un-checking both `Number of seconds to wait` checkboxes:
```

File: setup/README.md
Comment (quisquous) [inline setup/README.md:67]: This doesn't have to be done in this PR (and this text was here before), but I wonder if we should not recommend to run ACT as admin anymore.

**Comment by xiashtra** [comment]
Can we move this along?  It's a pretty bad look to tell people not to follow other guides because they are out-of-date, when we don't have an "official" replacement to point them to instead.

**Comment by quisquous** [comment]
> Can we move this along? It's a pretty bad look to tell people not to follow other guides because they are out-of-date, when we don't have an "official" replacement to point them to instead.

I was waiting on a response from xp from my last comments a few months ago.

File: setup/README.md
Comment (xpdota) [reply setup/README.md:67]: IIRC default launcher still elevates after a patch?

Comment (quisquous) [reply setup/README.md:67]: Mmm, that's fair.  It still seems like bad practice to me to recommend people run it as admin, but maybe that's an ok compromise here.

**Review by quisquous — APPROVED** [review]

File: setup/README.md
Comment (quisquous) [reply setup/README.md]: This looks like it's been resolved, so for the sake of getting this landed I'm just going to merge anyway with this outstanding "change request".

**Pull request merged**
